src/main/java/minicraft/core/Game.java

```diff
@@ -59,6 +70,12 @@ public static void exitDisplay() {
 	static boolean running = true;
 	public static void quit() { running = false; }
 
+	public static final ReloadableResourceManager resourceManager = new ReloadableResourceManager();
+	public static ResourcePackManager resourcePackManager;
+	public static VanillaResourcePack defaultResourcePack;
```

Comment:
I wonder if putting all these in Game class is good as it might make it more messy. I think they should be handled with a dedicated class.

Reply:
These can be integrated into a resource manager.